rfc/training-content-delivery.md

```diff
@@ -0,0 +1,75 @@
+
+Because of this approach's integration with our existing workflow, we can easily achieve it by placing this content in a subdirectory off of `cockroachlabs.com`, using a similar approach to both `/docs` and `/blog`.
+
+# Motivation
+
+We've begun developing training curriculum but have not yet determined where it lives, or how to best deliver it to attendees. Because this approach could drastically alter the authoring process, it's best to determine how this is done before we make substantial headway into producing these deliverables.
+
+By the end of this RFC, there will be a plan for the method we plan to use for delivering training content.
+
+# Guide-level explanation
+
+### Elements
+
+CockroachDB training is comprised of 3 elements:
+
+- **Presentations** give the instructor a tool to make sure the class follows the outlined curriculum, as well as reinforce topics discussed for students with a quick summary to help them mentally model the content, as well as a canvas for visual aids.
+- **Labs** provide attendees a chance to work with CockroachDB, using the information they just learned in the presentation. This should include a mixture of reinforcement of content form the presentation and step-by-step instructions to accomplish a task directly related to the presentation.
+- **Demos** give the instructor the option to demonstrate something that might be difficult for attendees to quickly reproduce themselves, e.g. anomalous reads from something like Jepsen. These can be "staged" labs to help students understand that something is true or real without having to relate the conditions for it.
+
+### Delivery
+
+To deliver the material, each element has its own needs:
+
+- Presentations require there being only a single bit of text presented at a time. To achieve this, I recommend using Google Slides.
+- Labs require much more text, as well as other hypertext tools that we've already developed on `/docs`. To achieve this, I recommend using Markdown rendered into HTML through Jekyll.
+- Demos each have different needs, but are often broadcast in the classroom from the instructor's computer. I recommend either recording demos that can be shown on YouTube, or solving demos on a case-by-case basis.
+
+### Structure
+
+As a wrapper around all of this content, the training material is hosted on `cockroachlabs.com/training`. We currently do not have password-restricted access on this content, but may in the future.
+
+This site is written in Markdown and converted to HTML through Jekyll and has the following structure:
+
+`home/<day>/<module>/`
+
+That is:
+- 1 home page for all training material (if this training system developers further, it's possible to insert levels above this)
+- 1 page for each day, which contain links to its modules
```

Comment:
For now, sounds fine to group by day. But if we decide, down the road, to expose this content for self-service use, we may then want to reassess the "day"-based approach.

Reply:
An alternative would be grouping by categories like:
- Introduction
- Ops Basics
- Dev Basics
- Monitoring/Troubleshooting
- Maintenance